tools/nimweb.nim

```diff
@@ -276,18 +276,18 @@ proc buildDoc(c: var TConfigData, destPath: string) =
     commands = newSeq[string](len(c.doc) + len(c.srcdoc) + len(c.srcdoc2))
     i = 0
   for d in items(c.doc):
-    commands[i] = "nim rst2html $# --docSeeSrcUrl:$#/$#/$# -o:$# --index:on $#" %
-      [c.nimArgs, c.gitRepo, c.gitCommit, d.pathPart,
+    commands[i] = "nim rst2html $# --docSeeSrcUrl:$#/$# -o:$# --index:on $#" %
+      [c.nimArgs, c.gitRepo, c.gitCommit,
```

Comment:
The `d.pathPart` of the URL was getting doubled up, eg. `https://github.com/nim-lang/Nim/blob/devel/lib/impure/lib/impure/nre.nim`. Removing it here fixes that. I ran a tool to look for regressions (github 404s), and it was unable to find any.